api/python/provisioner/commands/deploy_vm.py

```diff
@@ -92,6 +94,50 @@
 class DeployVM(Deploy):
     input_type: Type[inputs.NoParams] = inputs.NoParams
     _run_args_type = run_args_type
+    setup_ctx: Optional[SetupCtx] = None
+
+    def set_pillar_data(self, setup_type):
+        minions_ids = ['srvnode-1']
+        disk = []
+
+        if setup_type != SetupType.SINGLE:
+            minions_ids.append('srvnode-2')
+
+        res = self._cmd_run(
+            "lsblk -ndp | grep disk | awk '{ print $1 }'",
+            targets=self._primary_id()
+        )
+        if res[self._primary_id()]:
+            disk = res[self._primary_id()].split("\n")
+
+        for minion_id in minions_ids:
+            if len(disk) > 1:
+                self._cmd_run(
+                    "provisioner pillar_set "
+                    f" cluster/{minion_id}/storage/metadata_device "
+                    f'[\\"{disk[-2]}\\"]',
+                    targets=self._primary_id()
+                )
+                self._cmd_run(
+                    "provisioner pillar_set "
+                    f" cluster/{minion_id}/storage/data_devices "
+                    f'[\\"{disk[-1]}\\"]',
+                    targets=self._primary_id()
+                )
+
+            self._cmd_run(
+                "provisioner pillar_set "
```

Comment:
two comments:
- looks complicated, better to use single quotes for all parts to avoid any side effects
- please use additional parenthesis:
```
                self._cmd_run(
                    (
                        'provisioner pillar_set '
                        f' cluster/{minion_id}/storage/metadata_device '
                        f'[\\"{disk[-2]}\\"]'
                    ),
                    targets=self._primary_id()
                )
```

Reply:
Done